Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
+int DigitCount = 0;
 try
 {
-    if (File.Exists(filePath))
+    using var reader = new StreamReader(filePath);
+
+    while (!reader.EndOfStream)
     {
-        var content = File.ReadAllText(filePath);
 
-        int letterCount = content.Count(char.IsLetter);
-        int digitCount = content.Count(char.IsDigit);   
+        var line = reader.ReadLine()!;
 
-        Console.WriteLine($"The file contains {letterCount} letters and {digitCount} digits.");
-    }
-    else
-    {
-        Console.WriteLine($"File not found at path: {filePath}");
-    }
+        foreach (var character in line)
+        {
+            var helper = new Helper();
+
+            var isCharacter = helper.IsCharacterAndIfINeedToCount(character);
+            var isDigit = helper.IsDigitAndIfItIsInTheList(character);
+
+            IncreaseLetter_CountIfTrue(letter_count, isCharacter);
+            helper.IncreaseDigitCount(DigitCount, isDigit);
+        }
+        
+        void IncreaseLetter_CountIfTrue(int letter_countt, bool isCharacter)
+        {
+            letter_count = isCharacter ? ++letter_count : letter_count;
+        }
+    }       
 }
-catch (Exception ex)
+catch (FileNotFoundException)
 {
-    Console.WriteLine($"An error occurred: {ex.Message}");
+    Console.WriteLine($"File was not found in provided path");
 }
 
+Console.WriteLine($"The file contains {letter_count} letters and {DigitCount} digits.");
+Console.ReadKey();
+
+Console.WriteLine($"The file contains {new Helper().GetResult()}");
```

Comment:
Pagal mano siūlomus pakeitimus, čia reikės irgi pakeisti, kad grąžintų sumą:
Console.WriteLine($"The file contains {new Helper().GetResult(DigitCount, letter_count)} symbols.");

Reply:
o reikia čia iš vis to new Helper ? iš vis ar reikia helper clases ?

Reply:
Šio projekto kontekste reikia new Helper, nes GetResult yra Helper klasėje. Šiame projekte Helper klasė padeda išsikelti funkcijas iš Program.cs file.

Reply:
O tai gal reiktu visą funcionalumą iškelt į kitą klasę ? dabar pusė vienoje claseje kitas kitoje ?

Reply:
Jeigu gerai supratau klausimą, tai mano mąstymu reikia stengtis kuo mažiau funkcijų palikti program.cs file. Geriau kuo daugiau jų išsikelti ir tik po to kviesti program.cs